src/Core/Style.js

```diff
@@ -0,0 +1,113 @@
+import { FEATURE_TYPES } from 'Core/Feature';
+
+function rgba2rgb(orig) {
+    if (orig && orig.includes('rgba')) {
+        const rgb = orig.replace(/\s/g, '').match(/(\d+),(\d+),(\d+),(\d\.\d)/i);
+        return (rgb && rgb.length === 5) ? { color: `rgb(${rgb[1]}, ${rgb[2]}, ${rgb[3]})`, opacity: rgb[4] } : { style: orig, opacity: 1.0 };
+    } else if (orig && orig.includes('hsla')) {
+        const hsl = orig.replace(/\s/g, '').match(/(\d+),(\d+)%,(\d+)%,(\d\.\d)/i);
```

Comment:
You need to add a wider range for the regex, see here https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#HSL_colors